tests/foreman/cli/test_repository.py

```diff
@@ -343,47 +368,43 @@ def test_positive_create_9(self):
         content_type = u'docker'
         new_repo = self._make_repository({
             u'name': u'busybox',
+            u'docker-upstream-name': u'busybox',
             u'url': DOCKER_REGISTRY_HUB,
             u'content-type': content_type,
         })
         # Assert that urls and content types matches data passed
-        self.assertEqual(
-            new_repo['url'],
-            DOCKER_REGISTRY_HUB,
-            "Expected URL {0} but received {1}".format(
-                DOCKER_REGISTRY_HUB, new_repo['url'])
-        )
-        self.assertEqual(
-            new_repo['content-type'],
-            content_type,
-            "Expected content type {0} but received {1}".format(
-                content_type, new_repo['content-type'])
-        )
+        self.assertEqual(new_repo['url'], DOCKER_REGISTRY_HUB)
+        self.assertEqual(new_repo['content-type'], content_type)
+        self.assertEqual(new_repo['name'], u'busybox')
 
-    @skip_if_bug_open('bugzilla', 1155237)
     @run_only_on('sat')
     @data(
-        u'sha1',
-        u'sha256',
+        gen_string('alpha', 15),
+        gen_string('alphanumeric', 15),
+        gen_string('numeric', 15),
+        gen_string('latin1', 15),
+        gen_string('utf8', 15),
+        gen_string('html', 15),
     )
-    def test_positive_create_10(self, checksum_type):
-        """@Test: Create a YUM repository with a checksum type
+    def test_positive_create_docker_repo_2(self, name):
+        """@Test: Create a Docker repository with a randon name.
 
         @Feature: Repository
 
-        @Assert: A YUM repository is created and contains the correct checksum
-        type
-
-        @BZ: 1155237
+        @Assert: Docker repository is created and contains correct values.
 
         """
-        content_type = u'yum'
-        repository = self._make_repository({
+        content_type = u'docker'
+        new_repo = self._make_repository({
+            u'name': name,
+            u'docker-upstream-name': u'busybox',
+            u'url': DOCKER_REGISTRY_HUB,
             u'content-type': content_type,
-            u'checksum-type': checksum_type,
         })
-        self.assertEqual(repository['content-type'], content_type)
-        self.assertEqual(repository['checksum-type'], checksum_type)
+        # Assert that urls, content types and name matches data passed
+        self.assertEqual(new_repo['url'], DOCKER_REGISTRY_HUB)
+        self.assertEqual(new_repo['content-type'], content_type)
+        self.assertEqual(new_repo['name'], name)
```

Comment:
@Ichimonji10 sadly the CLI does not show the `docker-upstream-name` field and @elyezer had filed a BZ for it if I'm not mistaken...